_community_partners/gdg_mysuru.md

```diff
@@ -0,0 +1,5 @@
+---
+title : "GDG Mysuru"
+logo : "assets/images/community_partners/GDG_Mysuru.png"
+twitter : "GdgMysuru"
+---
```

Comment:
Do they have a website? If yes please add the website link.

If not then add any relevant link(Twitter/Meetup/LinkedIn) which can give more info about the community using website attribute.

Reply:
Okay

Reply:
done